puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/release/hammer-cli-katello-nightly-release.yaml

```diff
@@ -3,7 +3,7 @@
     project-type: pipeline
     sandbox: false
     triggers:
-      - timed: '0 22 * * *'
+      - github
```

Comment:
I think for this to work we need to specify something like -- 

```
    properties:
      - github:
          url: https://github.com/Katello/hammer-cli-katello
```

Reply:
That isn't necessary. The [docs](https://docs.openstack.org/infra/jenkins-job-builder/triggers.html#triggers.github) don't say anything about setting a `github` property.

As a more applicable example [rubygem-katello-nightly-release](http://ci.theforeman.org/job/rubygem-katello-nightly-release/) has been running as expected since its groovy conversion [occurred](https://github.com/theforeman/foreman-infra/pull/689) on 16 June.

There have been five builds triggered since the 16th.

Reply:
Fair, the Github plugin docs make me think it's required -- https://wiki.jenkins.io/display/JENKINS/GitHub+Plugin (Hyperlinks Between Pages section) but mus tbe just for that feature.